content/stopping-systemd-in-openshift.md

```diff
@@ -174,91 +182,114 @@ Detected architecture x86-64.
 ```
 
 We can see that systemd does not begin the stop sequence as was the
-case with podman.  This is because Kubernetes does not use the `STOPSIGNAL`
-instruction specified in the Dockerfile. To work around this situation we will
+case with podman.  This is because Openshift did not translate the
+`STOPSIGNAL` instruction specified in the Dockerfile (this will be fixed
+at Openshift 4.10). To work around this situation we will
 use [container lifecycle hooks](https://kubernetes.io/docs/concepts/containers/container-lifecycle-hooks/),
 to explicitly send `SIGRTMIN+3` to PID 1 (systemd).
 
 # Trying more isolated
 
 Let's see if this happens only for `SIGRTMIN+3` or for any signal
 specified via the `STOPSIGNAL` instruction. To investigate that, we
-will use the following Dockerfile:
+will use the following `Dockerfile.stopsignal-demo` Dockerfile:
 
 ```Dockerfile
 FROM quay.io/fedora/fedora:35
 COPY demo-signal.sh /demo-signal.sh
+RUN chmod a+x /demo-signal.sh
 STOPSIGNAL SIGINT
-ENTRYPOINT ["/demo-signal.sh"]
+CMD ["/demo-signal.sh"]
 ```
 
-The `demo-signal.sh` should have execute permission.  The content is:
+The `demo-signal.sh` should have executing permission. The content is:
```

Comment:
I think the previous text "execute permission" is more natural.

Reply:
thanks!